src/TotalCalculator.js

```diff
@@ -0,0 +1,39 @@
+var TotalCalculator = function(detailsList, orderedItems, subTotal) {
+  this.detailsList = hipsterCoffeeDetails;
+  this.orderedItems = orderedItems;
+  this.subTotal = subTotal;
```

Comment:
You could move the calculation of this in here, which makes sense because this is called totalCalculator